### PR TITLE
Move dotnet version to hardcoded version to workaround TFM change

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

**Review by nattress — APPROVED** [review]
The crossgen part looks good. Thanks!

File: eng/pipelines/coreclr/perf.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ trigger:
     - SECURITY.md
     - THIRD-PARTY-NOTICES.TXT
   
-pr: none
+#pr: none
```
Comment (am11) [inline eng/pipelines/coreclr/perf.yml:22]: @DrewScoggins, was it intentional to merge it with this line commented out? perf pipeline is triggering in new PRs.

Comment (DrewScoggins) [reply eng/pipelines/coreclr/perf.yml:22]: No this was an accident, sorry about that.

**Pull request merged**